ocamldoc/odoc_ast.ml

```diff
@@ -1021,7 +1021,7 @@ module Analyser =
 
     (** Analysis of a parse tree structure with a typed tree, to return module elements.*)
     let rec analyse_structure env current_module_name last_pos pos_limit parsetree typedtree =
-      print_DEBUG "Odoc_ast:analyse_struture";
+      print_DEBUG "Odoc_ast:analyse_structure";
```

Comment:
Sorry, I was remembering your corrections from the UK english "optimise" to "optimize" (for instance in bytecomp/simplif.ml), as it would make sense to have a coherent choice of english dialect. 

However, I forgot that ocamldoc use analyse everywhere (and tend to be quite french in term of english structure). So my comment is irrelevant, sorry.